charts/kong/templates/_helpers.tpl

```diff
@@ -409,7 +409,11 @@ The name of the service used for the ingress controller's validation webhook
 {{- if .Values.ingressController.admissionWebhook.enabled }}
 - name: webhook-cert
   secret:
+    {{- if not .Values.ingressController.admissionWebhook.certificate.provided }}
     secretName: {{ template "kong.fullname" . }}-validation-webhook-keypair
+    {{- else }}
+    secretName: {{ .Values.ingressController.admissionWebhook.certificate.secretName }}
+    {{- end }}  
```

Comment:
nit: it's a small thing, but unless there's a reason to check the negative, I feel like we should just make this a base if:
```suggestion
    {{- if .Values.ingressController.admissionWebhook.certificate.provided }}
    secretName: {{ .Values.ingressController.admissionWebhook.certificate.secretName }}
    {{- else }}
    secretName: {{ template "kong.fullname" . }}-validation-webhook-keypair
    {{- end }}  
```

Reply:
Agreed !